db/seeds.rb

```diff
@@ -51,7 +51,7 @@
   },
   {
     application: "LTI Starter App",
-    tenant: "starter-app",
+    tenant: Rails.application.secrets.default_lti_key,
```

Comment:
I'm not sure this change is good. I think this should remain a string in the seeds. The forked apps use strings to list the tenants too. There really isn't any reason to put them in the secrets. Since you can't change it without changing for everyone.